PROTOCOL.md

```diff
@@ -100,6 +103,19 @@ By default, the reference implementation stores data files in directories that a
 This directory format is only used to follow existing conventions and is not required by the protocol.
 Actual partition values for a file must be read from the transaction log.
 
+### Change Data Files
+Change data files are stored in a directory at the root of the table named `_change_data`, and represent the changes for the table version they are in. For data with partition values, change data files are stored within the `_change_data` directory in their respective partitions (i.e. `_change_data/part1=value1/...`). Writers can _optionally_ produce these change data files as a consequence of operations that change underlying data, like `UPDATE`, `DELETE`, and `MERGE` operations to a Delta Lake table. Operations that only add new data should not produce separate change files. When available, change data readers should use the change data files instead of computing changes from the underlying data files.
+
+In addition to the data columns, change data files contain additional columns that identify the type of change event:
+
+Field Name | Data Type | Description
+-|-|-
+_change_type|`String`| `insert`, `update_preimage` , `update_postimage`, `delete` __(1)__
+_commit_version|`Long`| The Delta log or table version containing the change.
+_commit_timestamp|`Timestamp`| The timestamp associated when the commit was created.
```

Comment:
These two columns are not in data change files.

Reply:
Should we still mention them somewhere since they are inferred by the reader at runtime? It's not clear if this should be an actual requirement of readers...